integration_tests/cork_test.go

```diff
@@ -117,292 +114,3 @@ func (s *IntegrationTestSuite) getCurrentCount() (*sdk.Int, error) {
 
 	return &count, nil
 }
-
-func (s *IntegrationTestSuite) TestCork() {
```

Comment:
Well, just like I said in sprint, we have a PR where we'll address everything concerning the tests (unit and integration). The only reason we touched that bit, is because it'll cause an error in the CI, since we don't have the initial cork structure.